tests/utils/index.js

```diff
@@ -1621,4 +1639,5 @@ module.exports = {
   isK3D,
   stopCouchDb,
   startCouchDb,
+  getDefaultForms,
```

Comment:
haha, I had a slur of e2e tests failing. At some point some test couldn't select a form because it was obscured by other forms listed in the popup. So I changed how test cleanup is handled to only keep default forms around and delete ephemeral ones. 

Do you think this is important enough to move to another PR?